packages/homepage/content/docs/4-explore.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Explore Page & Curators"
+authors: ["CompuIves"]
+description: "Curators can pick outstanding sandboxes to be featured on the explore page. You can find info here on how to get picked or how to become a curator."
+---
+
+## Explore Page
+
+CodeSandbox features an [explore page](/explore). The explore page functions as a place where curators can highlight interesting or visually appealing sandboxes created by others.
+
+## What is a curator
+
+A curator can select sandboxes that end up on the explore page. You can get an invitation to become a "curator" from the staff or other curators. Everyone is applicable to become a curator, but we generally select people that are already active on CodeSandbox or are knowledgeable in a specific field (eg. animations).
+
+If you're interested in becoming a curator you can send us an e-mail at <mailto:curators@codesandbox.io>.
+
+## Which sandboxes get picked
+
+We can pick sandboxes based on their contents and their appearance. This can be exclusive, meaning that we can pick sandboxes that are not visually appealing but show some really interesting logic or approach. We can also pick sandboxes that only show something really good visually. A nice visual is always a plus, but not a requirement.
```

Comment:
```suggestion
Sandboxes get picked based on their contents and their appearance. This can be exclusive, meaning that we can pick sandboxes that are not visually appealing but show some really interesting logic or approach. We can also pick sandboxes that only show something really good visually. A nice visual is always a plus, but not a requirement.
```